app/src/main/res/values-ar/strings.xml

```diff
@@ -178,7 +178,6 @@
   <string name="size_gb">%s ج.ب</string>
   <string name="correct">صحيح!</string>
   <string name="topic_prefix">الموضوع: %s</string>
-  <string name="chapter_count_with_story_name">%1$s في %2$s</string>
   <plurals name="chapter_count">
```

Comment:
Shouldn't this one also be removed everywhere if it's no longer being used?

Reply:
Hey Ben. I believe it is marked as deleted. I have cross-checked on my fork and on the code itself on my machine and both don't have that line.

Reply:
Just to confirm, you're talking about ``chapter_count`` and not ``chapter_count_with_story_name``? Per GitHub, it's very much still present on your remote feature branch.

Also, it seems to be on your fork, too: https://github.com/kkmurerwa/oppia-android/blob/eebece9e8887db048d9db72e0d8c104a47ce1132/app/src/main/res/values-ar/strings.xml#L181.

Reply:
Hey. I was under the impression that we are talking about `chapter_count_with_story_name` and not `chapter_count`. `chapter_count` seems to have been made redundant as well so I will work on removing it and submitting the changes.

Reply:
Update on this. I looked at the code in more detail and I found one instance in which `chapter_count` is still being used. The `computeChapterCountText()` method on line 44 of the `StorySummaryViewModel` still uses the string. I have referenced it in a separate comment.

Reply:
Ah okay, that makes sense. Thanks for checking!